docs/execution-environments/r-support.rst

```diff
@@ -1,178 +1,278 @@
 
-        Sys.setenv('NEPTUNE_API_TOKEN'='eyJhcGlfYWRkcmVzcyI6Imh0dHBz')
+* Have ``Python 3.x`` installed. The easiest way to |install Python environment is with Conda|.
 
-    .. code:: R
+.. tip::
 
-        init_neptune(project_name = 'my_workspace/my_project',
-                     api_token = Sys.getenv('NEPTUNE_API_TOKEN')
-                     )
+    If you are using R studio you can just follow along and you will be prompted by the console to install miniconda.
 
-**Specifying Python environment**
+Step 1: Install Neptune package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Install ``neptune`` package for R from CRAN.
 
-* Python
+.. code-block:: R
 
-    .. code:: R
+    install.packages('neptune', dependencies = TRUE)
 
-        init_neptune(project_name = 'shared/r-integration',
-                     api_token = 'ANONYMOUS'
-                     python='python',
-                     python_path='/usr/bin/python3')
+.. note::
 
-* venv
+    See :ref:`neptune client for R installation guide <installation-neptune-r>`.
 
-    .. code:: R
+Step 2: Import neptune package and initialize Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Add the following snippet at the top of your script.
 
-        init_neptune(project_name = 'shared/r-integration',
-                     api_token = 'ANONYMOUS'
-                     python='venv',
-                     python_path='my_venv')
+.. code-block:: R
 
-* conda
+    library(neptune)
 
-    .. code:: R
+    init_neptune(project_name = 'shared/r-integration', api_token = 'ANONYMOUS')
 
-        init_neptune(project_name = 'shared/r-integration',
-                     api_token = 'ANONYMOUS'
-                     python='conda',
-                     python_path='my_conda_env')
+.. tip::
 
-* miniconda
+    You can also use your personal API token. Read more about how to :ref:`find your Neptune API token <how-to-setup-api-token>`.
 
-    .. code:: R
+.. note::
 
-        init_neptune(project_name = 'shared/r-integration',
-                     api_token = 'ANONYMOUS'
-                     python='miniconda',
-                     python_path='my_miniconda_env')
+    This will use your default Python. If you are hitting ``Error: could not find a Python environment`` error or you want to use some other Python version you have on your system :ref:`jump to this section and read how<r-non-default-python>`.
+
+Step 3: Create an experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below to create a Neptune experiment:
+
+.. code-block:: R
+
+    create_experiment(name='minimal example')
+
+This also creates a link to the experiment. Open the link in a new tab.
+The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
+
+.. note::
+
+    Neptune logs your hardware consumption and console outputs automatically.
+
+    Go to the ``Monitoring`` in your Neptune experiment to see it.
```

Comment:
If the user does not have psutils installed, monitoring tab will be empty, maybe properties tab?

Reply:
Or at least mention that user has to have it installed to see this

Reply:
It actually installs it automatically, or maybe just R studio does -> I'll move it to other options just to make sure.